m/base.go

```diff
@@ -14,23 +14,25 @@ func PrintFlush(targetMessage string) {
 }
 
 // Get the linked tile at the specified address
-func GetLink[A integer](address A) Link {
+func GetLink[A integer](address A) UnspecifiedBuilding {
 	return nil
 }
 
 // Retrieve a list of units that match specified conditions
 //
 // Conditions are combined using an `and` operation
-func Radar(from Ranged, target1 RadarTarget, target2 RadarTarget, target3 RadarTarget, sortOrder bool, sort RadarSort) Unit {
+func Radar(from Building, target1 RadarTarget, target2 RadarTarget, target3 RadarTarget, sortOrder bool, sort RadarSort) Unit {
```

Comment:
Please do not rename objects whose names are taken from internal game names: https://github.com/Anuken/Mindustry/blob/master/core/src/mindustry/logic/LExecutor.java#L714

This applies to all other object names that you have renamed. I wish to keep the naming as close as possible to in-game naming so that future maintenance would be easier.

Reply:
I fixed it now and used the names from Minustry

I am not completely sure how the typing there works, like what are the "c" postfix classes